scripts/inner/kubestellar-kube-bind

```diff
@@ -114,7 +108,7 @@ kubectl apply -f - <<EOF
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: kbmap-$kcp_id
+  name: kbmap-$space_name
```

Comment:
Does not have to be done in this PR, but this convention for representing the mapping should be documented.